DatadogRUM/Tests/Instrumentation/Views/RUMViewsHandlerTests.swift

```diff
@@ -28,8 +28,7 @@ class RUMViewsHandlerTests: XCTestCase {
 
     func testGivenAcceptingPredicate_whenViewDidAppear_itStartsRUMView() throws {
         let viewName: String = .mockRandom()
-        let viewControllerClassName: String = .mockRandom()
-        let view = createMockView(viewControllerClassName: viewControllerClassName)
+        let view = ViewControllerMock()
```

Comment:
question/ I’m not entirely clear on this change or if it’s missing the removal of the `createMockView()` implementation. From the PR description, I understand that [`createMockView()`](https://github.com/DataDog/dd-sdk-ios/blob/832f47dd8e9df17880750aa50b49a5f04e277d48/DatadogRUM/Tests/Mocks/RUMFeatureMocks.swift#L913-L928) is causing issues in Xcode 16. Ideally, I would expect to see that method either fixed or completely replaced with an alternative approach. While we’ve replaced it in a few places, the original problematic method still appears to be present.